packages/gen-ai-hub/src/index.ts

```diff
@@ -6,4 +6,11 @@ export {
   OpenAiChatCompletionOutput
 } from './client/index.js';
 export { CustomRequestConfig, BaseLlmParameters } from './core/index.js';
-export { GenAiHubClient } from './orchestration/index.js';
+export {
+  GenAiHubClient,
+  GenAiHubCompletionParameters,
+  GenAiHubCompletionResponse,
+  PromptOptions,
+  LlmConfig,
```

Comment:
We discussed about this, `PromptConfig` would be consistent with the other module names, but, the fields in the Prompt object (`messages_history`,`template`,`template_params`) felt more like options than something you would configure for how the module works. 

But, if you think consistency is better for usage, when we switch.

Reply:
nah I don't think it's that important, we can keep it at `PromptConfig`. Or maybe, we could just call it `Prompt`? Because it feels a bit awkward if you have prompt options, without having a "prompt" itself..

Reply:
I would stick to `PromptConfig` instead then.